sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/systemevents/IotHubDeviceConnectedEventData.java

```diff
@@ -6,6 +6,35 @@
 
 import com.azure.core.annotation.Immutable;
 
-/** The IotHubDeviceConnectedEventData model. */
+/** Event data for Microsoft.Devices.DeviceConnected event. */
 @Immutable
-public final class IotHubDeviceConnectedEventData extends DeviceConnectionStateEventProperties {}
+public final class IotHubDeviceConnectedEventData extends DeviceConnectionStateEventProperties {
```

Comment:
When models are regenerated, the existing files get overwritten if the new codegen has to write to the same file but if a file was no longer required but was still in the directory, it doesn't get deleted. So, make sure the files that were previously generated are deleted if they are no longer required.

Reply:
There are some new ones but no one is deleted. Most file changes have javadoc update.